ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -135,6 +142,40 @@ class MySqlNamedBlobDb implements NamedBlobDb {
   private static final String SOFT_DELETE_QUERY_V2 =
       String.format("UPDATE %s SET %s = ? WHERE %s", NAMED_BLOBS_V2, DELETED_TS, PK_MATCH_VERSION);
 
+  /**
+   * Pull the stale blobs that need to be cleaned up
```

Comment:
Thanks for adding the comments, I'm wondering is there any easy way to filter the staled named blob by only use where? Left join is always used to join two tables.

Reply:
Yeah, I thought about this, and can't think a better way than current query. I timed the query and it perform well. And since it's a background batch job (not used in user requests), it should be ok